src/engine/specification.md

```diff
@@ -219,7 +219,7 @@ This structure has the syntax of `ExecutionPayloadV1` and appends a single field
 - `baseFeePerGas`: `QUANTITY`, 256 Bits
 - `blockHash`: `DATA`, 32 Bytes
 - `transactions`: `Array of DATA` - Array of transaction objects, each object is a byte list (`DATA`) representing `TransactionType || TransactionPayload` or `LegacyTransaction` as defined in [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718)
-- `withdrawals`: `Array of WithdrawalV1` - Array of withdrawals, each object is an `OBJECT` containing the fields of a `WithdrawalV1` structure.
+- `withdrawals`: `Array of WithdrawalV1|null` - Array of withdrawals, each object is an `OBJECT` containing the fields of a `WithdrawalV1` structure.
```

Comment:
```suggestion
- `withdrawals`: `(Array of WithdrawalV1) | null` - Array of withdrawals, each object is an `OBJECT` containing the fields of a `WithdrawalV1` structure.
```

A nit: it may read as `null` values are allowed in the array.

Reply:
You think this is necessary? I wouldn't mind it's just that the way it is currently written is consistent with everywhere else in the document.

Reply:
I don't insist if it reads as intended for everyone